src/libraries/Common/src/System/Net/ContextAwareResult.Windows.cs

```diff
@@ -10,7 +10,6 @@ internal sealed partial class ContextAwareResult
     {
         private WindowsIdentity? _windowsIdentity;
 
-        // Security: We need an assert for a call into WindowsIdentity.GetCurrent.
```

Comment:
This doesn't seem related

Reply:
It's related in that I was auditing use of GetCurrent, this showed up in the search, and the comment is erroneous.